website/docs: Add instructions for installing RC versions (#18099)
goauthentik/authentik: website/docs/install-config/beta.mdx

---
title: Beta and release candidate versions
---

import TabItem from "@theme/TabItem";
import Tabs from "@theme/Tabs";

You can test upcoming authentik versions before they are released as stable. There are two types of pre-release versions available:

- **Release Candidates (RC)**: Near-final versions that are feature-complete and undergoing final testing before a stable release. RCs are more stable than Beta versions and are recommended for testing in staging environments.

> - **Beta versions**: Early access to major new features that are still in active development. Beta versions may have more bugs and breaking changes.

Reviewer: ```suggestion
- **Beta versions**: Early builds that include new, major, or experimental features and may introduce bugs or breaking changes.
```

I'd mabye cut "major":

```suggestion
- **Beta versions**: Early builds that include new, or experimental features and may introduce bugs or breaking changes.
```

Reviewer: It's not really an access, so "builds" feels better in my book

> It is recommended to upgrade to the latest stable release before upgrading to RC or Beta images.

Reviewer 2: ```suggestion
It is recommended to upgrade to the latest GA release before upgrading to RC or Beta images.
```

> It is recommended to upgrade to the latest stable release before upgrading to RC or Beta images.

Reviewer: also images -> version to be more general and not k8s/docker specific and make it clear that its really the version.

:::warning
Downgrading from RC or Beta versions is not supported. It is recommended to take a backup before upgrading, or test these versions on a separate install. Upgrading from RC or Beta versions to the next stable release is usually possible, however also not supported.
:::

## Release Candidate (RC) versions

> Release Candidates are available for testing before major version releases. To use an RC version, you need to specify the exact RC tag (e.g., `2025.10.0-rc3`).

Reviewer 2: ```suggestion
Release candidates are available for testing before major version releases. To use an RC version, you need to specify the exact RC tag (e.g., `2025.10.0-rc3`).
```

> Release candidates are available for testing before major version releases. To use an RC version, you need to specify the exact RC tag (e.g., `2025.10.0-rc3`).

Reviewer: > before major version releases

This would make sense considering our usage of semver, but I feel like it's generally thought that the major version would be the x in x.y.z, so the year in our case. Do we use "major version" elsewhere or is there anything we can do to clarify this?

<Tabs
  defaultValue="docker-compose"
  values={[
    {label: 'Docker Compose', value: 'docker-compose'},
    {label: 'Kubernetes', value: 'kubernetes'},
  ]}
  groupId="platform">
  <TabItem value="docker-compose">

> Add the following block to your `.env` file, replacing `2025.10.0-rc3` with the desired RC version:

Reviewer 3: If the third symbol is always going to be zero, describe the pattern accurately without aping the KBD setting, using the full range of available emphasis:

> RC tags follow the format **YYYY**.**MM**.0-rc**N**.

---
title: Beta and Release Candidate versions
---

You can test upcoming authentik versions before they are released as stable. There are two types of pre-release versions available:

> - **Release Candidates (RC)**: Near-final versions that are feature-complete and undergoing final testing before a stable release. RCs are more stable than Beta versions and are recommended for testing in staging environments.

Reviewer 2: ```suggestion
- **Release Candidates (RC)**: Near-final versions that are feature-complete and undergoing final testing before a GA release. RCs are more stable than Beta versions and are recommended for testing in staging environments.
```
